doc/articles/features/dialogs.md

```diff
@@ -30,6 +30,21 @@ ContentDialogResult result = await noWifiDialog.ShowAsync();
 
 It is crucial to set the `XamlRoot` property before calling `ShowAsync`. This way the dialog will become asssociated with the visual tree. `XamlRoot` can be retrieved from any loaded control in your window (e.g. a `Button`, your `Page`, etc.).
 
+## Customize a `ContentDialog`
+
+You can use various properties to customize its display and also provide a custom XAML-based content for it. For more information, see [ContentDialog Class | Microsoft Learn](https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.contentdialog).
+
+In order for your custom dialog to follow the Fluent design, it's important that you add the following line in the constructor of your dialog:
+
+```csharp
+Style = Application.Current.Resources["DefaultContentDialogStyle"] as Style;
+```
+You dialog will look similar to this dialog:
+<img width="273" alt="WithStyle" src="https://github.com/unoplatform/uno/assets/1608424/ca1c3dd8-694e-4149-8778-cbf103098fd3">
+
+If you omit this line, your dialog will look like this:
+<img width="281" alt="NoStyle" src="https://github.com/unoplatform/uno/assets/1608424/e1066618-1d6b-4215-8156-f1cf21a30c9d">
```

Comment:
@ArchieCoder the images would need to be placed in the assets folder for them to display properly on the documentation website please
Here is an example: https://github.com/unoplatform/uno/blob/4fae8b53d71ad87ca0f84e0d603cda53c6bf205c/doc/articles/features/ElevatedView.md?plain=1#L37